tests/geom_timed_test.c

```diff
@@ -17,10 +17,11 @@
 #include <time.h>
 #ifdef _WIN32
 #include <windows.h> /* for time */
```

Comment:
This was just a change in case anyone wants to compile this test program on Windows (not used by us in GitHub Actions).